src/renderers/shaders/ShaderChunk/bumpmap_pars_fragment.glsl

```diff
@@ -34,6 +34,8 @@
 
 		float fDet = dot( vSigmaX, R1 );
 
+		fDet *= ( float( gl_FrontFacing ) * 2.0 - 1.0 );
```

Comment:
Is this intentional? Should it just be...

```js
float fDet = dot( vSigmaX, R1 ) *  float( gl_FrontFacing );
```

Reply:
What I have coded was intentional and works in my tests. It is equivalent to a sign flip if back-facing.

Reply:
Ah, I see. This code multiplies `fDet` by `1.0` or `-1.0`. Looks good!